bootstrap-templates/default-20021118/common/providers.tf

```diff
@@ -0,0 +1,18 @@
+provider "aws" {
+  region              = "us-east-2"
```

Comment:
This will need to be parameterized per-region

Reply:
actually i lied. this one is fine. if i add an aws provider to `deploy/providers.tf`, it will need to be parameterized.